source/use-cases/client-side-field-level-encryption-guide.txt

```diff
@@ -158,12 +160,22 @@ compliant to data privacy regulations with MongoDB.
 Implementation
 --------------
 
-This guide is meant for users new to Client-Side Field Level Encryption (CSFLE).
-It covers the why and how at a high level, and is not meant to produce working
-code in a single file.
+This section explains the following configuration and implementation
+details of CSFLE:
+
+- Software required to run your client and server in your local development
+  environment.
+
+- Creation, setup, and validation of the encryption keys.
+
+- Configuration of the client for automatic field-level encryption.
+
+- Demonstration of the client writing, querying, and reading encrypted
+  fields.
```

Comment:
nit: "Demonstration" is not a "configuration and implementation detail"

Maybe: "How to write, read, and query on encrypted fields"

Reply:
Good catch, will update.